db/migrations/1_create_students_table.rb

```diff
@@ -3,20 +3,27 @@
 db = SQLite3::Database.new "./db/dev.db"
 
 db.execute "
-  create table students (
+  create table bands (
     id INTEGER PRIMARY KEY ASC,
     name VARCHAR(255),
-    email VARCHAR(255),
-    discord VARCHAR(255)
+    firstalbumyear VARCHAR(255),
+    myrank VARCHAR(255)
   );
 "
 
-students = [
-  ["Chad Ostrowski", "ostrowski@stevens.edu", "ProfessorO"],
+bands = [
+  ["Led Zeppelin", "1968", "1"],
+  ["Steely Dan", "1972", "2"],
+  ["Pink Floyd", "1967", "3"],
+  ["The Beatles", "1963", "4"],
+  ["Rush", "1974", "5"],
+  ["Creedence Clearwater Revival", "1968", "6"],
+  ["Iron Maiden", "1981", "7"]
 ]
 
-students.each do |student|
+students.each do |band|
```

Comment:
Did this work? You renamed the variable from `students` to `bands` on line 14 above, but here you did not make a corresponding change. I think Ruby will complain that `students` is undefined. Let me know if that's what happens when you run `ruby db/migrate.rb`